packages/dictionary/data/chords.json

```diff
@@ -85,25 +79,25 @@
   "9b5": ["1P 3M 5d 7m 9M"],
   "9no5": ["1P 3M 7m 9M"],
   "9sus4": ["1P 4P 5P 7m 9M", ["9sus"]],
-  "m": ["1P 3m 5P"],
+  "m": ["1P 3m 5P", ["minor", "min"]],
```

Comment:
I'm reluctant to add "minor" as chord name because that name is usually associated to scales.